src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -24,49 +27,69 @@
 import java.util.Set;
 import java.util.TreeSet;
 
+/**
+ * Provides configuration information through http API service
+ */
 public class ConfigServer {
 
+  public final Log log = LogFactory.getLog(ConfigServer.class);
+
   private Server server;
   private Engine engine;
-  private SchedulerConf schedulerConf;
+  private HdfsFrameworkConfig hdfsConf;
```

Comment:
Don't think I like calling this hdfsConf, since it's really the frameworkConf (mesos-site.xml), not the actual hdfs config files (hdfs-site.xml, core-site.xml)